tests/integration/testmarketplace.nim

```diff
@@ -5,22 +5,37 @@ import ./marketplacesuite
 import ./twonodes
 import ./nodeconfigs
 
-twonodessuite "Marketplace", debug1 = false, debug2 = false:
+multinodesuite "Marketplace":
+  let marketplaceConfig = NodeConfigs(
+    clients: CodexConfigs.init(nodes=1).some,
+    providers: CodexConfigs.init(nodes=1).some,
+  )
```

Comment:
Why not use `marketplacesuite` here (and in others below)?

Reply:
I changed it. It makes very little difference technically, but makes slightly more sense. :D